### PR TITLE
Machine

### PR DESCRIPTION
**Pull request by VolodymyrBarik**

### CODE REVIEW

File: Bulldozer.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("The bulldozer has started to work!");
+    }
```
Comment (Nick97-git) [inline Bulldozer.java:4]: add empty line between methods, the same for other classes

File: Machine.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+abstract public class Machine {
```
Comment (Nick97-git) [inline Machine.java:1]:
```suggestion
 public abstract class Machine {
```

File: Main.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+public class Main {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+
```
Comment (Nick97-git) [inline Main.java:6]: remove empty line

File: Main.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        Machine[] machines = {truck, bulldozer, excavator};
+        for(Machine unit: machines) {
```
Comment (Nick97-git) [inline Main.java:8]:
```suggestion
        for(Machine machine: machines) {
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]

**Review by Nick97-git — CHANGES_REQUESTED** [review]
left one more comment + files on the screenshot shouldn't be here
Screenshot 2022-06-21 at 11 38 33

File: Bulldozer.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+public class Bulldozer extends Machine {
```
Comment (Nick97-git) [inline Bulldozer.java:1]: where si `package core.basesyntax;`? the same for other classes

File: Bulldozer.java [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
```
Comment (Nick97-git) [inline Bulldozer.java:2]: remove empty line, the same for other classes

**Review by Nick97-git — CHANGES_REQUESTED** [review]

File: Bulldozer.java (hunk 5 above)
Comment (VolodymyrBarik) [reply Bulldozer.java:1]: What is core.basesyntax; ? Where can I read about that? Should I import this package, or it is about doing syntax right?

**Review by Nick97-git — CHANGES_REQUESTED** [review]
empty lines in every class 
Screenshot 2022-06-24 at 13 48 28

**Review by Nick97-git — APPROVED** [review]

**Pull request closed without merging**